tree-construction/tests20.dat

```diff
@@ -514,3 +514,57 @@
 |       <math annotation-xml>
 |         encoding=" text/html "
 |     <div>
+
+#data
+<math><annotation-xml> </annotation-xml>
+#errors
```

Comment:
Yeah, we should. But AFAIK the errors in general are frequently wrong and I felt like giving up when I did this.